src/app/address/controllers/addressSearch.js

```diff
@@ -6,19 +6,24 @@ const {
 
 class AddressSearchController extends BaseController {
   async saveValues(req, res, callback) {
+    const addressPostcode = req.body["address-search"];
+
     try {
       const addressPostcode = req.body["address-search"];
       const searchResults = await this.search(
         req.ordnanceAxios,
         addressPostcode
       );
       super.saveValues(req, res, () => {
+        req.sessionModel.set("isSuccessful", true);
```

Comment:
Once we have the data model, we might know about properties we want to store that aren't user accessible (e.g. UPRN). Even so, it might be cleaner to have a clear "request was successful" property 👍

Reply:
@merlinc I agree, for now I can change it to be more explicit.

Reply:
My bad! A more explicit request boolean value would translate nicer anyway